test/automated/reflection/target_method_predicate.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../automated_init'
+
+context "Reflect" do
+  context "Reflection" do
+    context "Target Method Predicate" do
+      context "Method Implemented" do
+        subject = Reflect::Controls::Subject.example
+        reflection = Reflect.(subject, :SomeConstant, strict: true)
```

Comment:
Consider: don't supply `strict: true` since it has no bearing on the test

Reply:
Consider: `subject` and `reflection` can be assigned outside this context, and shared between the `Method Implemented` and `Method Is Not Implemented` context